content/en/tracing/guide/trace_queries_dataset.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How Trace Queries Source Data
+kind: guide
+further_reading:
+    - link: "/tracing/trace_explorer/trace_queries/"
+      tag: "Documentation"
+      text: "Trace Queries"
+---
+
+## Overview
+ 
+With Trace Queries, you can find entire traces based on the properties of multiple spans and the relationships between those spans within the structure of the trace. To learn more, read the [Trace Queries documentation][1].
+
+{{< img src="tracing/trace_queries/trace_queries.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Queries UI" >}}
+
+## How Trace Queries source data
+
+Datadog uses the [Intelligent Retention Filter][6] to index data for Trace Queries. It does so by performing: 
+
+- [Flat sampling](#1-flat-sampling): A uniform 1% sample of ingested spans.
+- [Diversity sampling](#diversity-sampling): A representative, diverse selection of traces to keep visibility over each environment, service, operation, and resource.
+
+{{< img src="tracing/trace_queries/trace_queries_new_dataset.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="1% Flat Sampling & Diversity Sampling" >}}
+
+**Note**: Spans indexed by flat sampling and diversity sampling do not count towards the usage of indexed spans, and therefore, **do not impact your bill**.
+
+### 1% flat sampling
+
+Flat 1% sampling is applied based on the `trace_id`, meaning that all spans belonging to the same trace share the same sampling decision. To learn more, read the [one percent flat sampling documentation][2].
+
+### Diversity sampling
+
+Every 15 minutes, diversity sampling retains at least one span and the associated trace for each combination of environment, service, operation, and resource. This occurs for the `p75`, `p90`, and `p95` percentile of latencies to ensure that you can always find example traces in service and resource pages, even for low traffic endpoints. To learn more, read the [diversity sampling documentation][3].
+
+## Impact of enabling Trace Queries
+
+When you enable Trace Queries on your account, the Intelligent Retention filter starts to index more data.
+
+You can query spans indexed by the Intelligent Retention filter in the [Trace Explorer][4]. As a result, you might notice a spike in the number of indexed spans in Trace Explorer queries. This change is indicated by an event overlay showing an **Intelligent Retention Filter change** event.
+
+To find spans that are sampled by the 1% flat sampling or the diversity sampling methods, add a `retained_by:(flat_sampled OR diversity_sampling)` query parameter in the Trace Explorer.
+
+_[add trace explorer screenshot]_
```

Comment:
Note: This still says "add trace explorer screenshot". Is there supposed to be a screenshot?

Reply:
I'll add the screenshot now